### DISCUSSION TITLE
Cannot install foundry due to SSL certificate expiration.

### DISCUSSION THREAD

**Discussion by DelfKo** [Cyfrin/foundry-full-course-cu#783] [General]
GM everyone, I hope you guys are doing alright.

I cannot install Foundry on my VSCode using the command : "curl -L https://foundry.paradigm.xyz | bash".
I also used the "curl -L https://foundry.paradigm.xyz | bash -k (--insecure)" version.

This is the error message that I get : 

curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html 
[…]

I went to the website page but as a native in web dev, I don't really understand what is in this page. 

Can you please help me debug my situation please ?

**Comment by theirrationalone** [comment]
Please Update your system and its drivers.
